### ISSUE TITLE
Dynamic Model Registering not working with Instant Run and specific Gradle version.

### ISSUE THREAD

**Issue by r-mckay** [Odoo-mobile/framework#258]
Hello.

The commit 510762725fef7a3fe789a9c826fed30a20d04784 introduces
`Each of the model class dynamically register when app starts with ModelRegistryUtils`

However, this feature is not working properly with Instant Run and Gradle.
Please see: http://stackoverflow.com/questions/36572515/dexfile-in-2-0-versions-of-android-studio-and-gradle/36594966#36594966

Disabling Instant Run solves this and models seem to register correctly.

Thanks.

**Comment by dpr-odoo** [comment]
@r-mckay 
Yes, For now developer need to **disable Instant Run for this framework**. We will fix this issue for *Instant Run* soon for debug mode.

Thanks.

**Comment by mikedream89** [comment]
Hi guys， this issue already be fixed?

**Comment by Ana-bytes** [comment]
same question here, is this fixed?
